tmva/sofie/inc/TMVA/SOFIE_common.hxx

```diff
@@ -105,6 +105,7 @@ ETensorType GetTemplatedType(T /*obj*/ ){
 namespace UTILITY{
 template<typename T>
 T* Unidirectional_broadcast(const T* original_data, const std::vector<size_t> original_shape, const std::vector<size_t> target_shape);
+std::vector<size_t> Multidirectional_broadcast(const std::vector<size_t> input1_shape, const std::vector<size_t> input2_shape);
```

Comment:
since we pass the input tensor by values, we change them inside the function adding 1's,  the `const` should be removed

Reply:
Done.